Automattic/jetpack
# Chart Library: Add Grid

## Proposed changes:

This PR introduces a new grid control component 

**Changes Made:**

**Dependencies:**
- Added @visx/grid to dependencies.

**Component:**
- Created a new `grid control` component

**Storybook:**
- Adds a new storybook + stories to demonstrate usage of grid
- Adds the new grid component to the barchart and makes it available on all

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Run `pnpm install` as dependencies have been added
* In your Jetpack local env, navigate to the Storybook development environment:

   ```
  pnpm i
  cd projects/js-packages/storybook
  pnpm run storybook:dev
  ```

* Find `JS Packages/Charts/Grid Control` in the sidebar

* Verify the grids:
  - Renders correctly with sample data
  - Responds to different data sets via Storybook controls `x`,`y`,`xy`, and `none`

* Find `JS Packages/Charts/Bar Chart` in the sidebar
  - check the bar chart renders with the new grid controls
